zh/option/series/treemap.md

```diff
@@ -596,24 +596,6 @@ treemap 默认把第一个维度（Array 第一项）映射到『面积』上。
 ) }}
 {{ /if }}
 
-#${prefix} colorAlpha(Array) = null
```

Comment:
This colorAlpha doesn't need to be removed. It's not same with `itemStyle.colorAlpha`

The related doc is https://echarts.apache.org/zh/option.html#series-treemap.colorAlpha

Reply:
As is tested, this "global" colorAlpha is not working either. My test code is as following:
<details>
<summary>Test Code</summary>

The second red rectangle is not inheriting parent node's `colorAlpha`. 
```
option = {
  series: [
    {
      type: 'treemap',
      label: {
        color: 'black',
      },
      colorAlpha:0.3,
      data: [
        {
          name: 'nodeA',
          value: 10,
          color: ['rgba(255,0,0,0.3)', 'red'],
          children: [
            {
              name: 'nodeAa',
              value: 4,
            },
            {
              name: 'nodeAb',
              value: 6,
            },
          ],
        },
        {
          name: 'nodeB',
          value: 20,
          children: [
            {
              name: 'nodeBa',
              value: 20,
              children: [
                {
                  name: 'nodeBa1',
                  value: 20,
                },
              ],
            },
          ],
        },
      ],
    },
  ],
};
```
</details>

Since it's not working properly, I am not sure if we should delete it in doc too or should we take it as another issue. Developers can set rgba in each node, but it might be troublesome when there are dozens to set.

Reply:
`colorAlpha` here should be an array to represent the mapping range from value. e,g: `colorAlpha: [0, 1]`

Reply:
Sorry I didn't read the documentation carefully, will revise my PR.